api/types/constants.go

```diff
@@ -760,6 +760,10 @@ const (
 	// created from the AWS OIDC Integration.
 	OriginIntegrationAWSOIDC = common.OriginIntegrationAWSOIDC
 
+	// OriginIntegrationAWSRolesAnywhere is an origin value indicating that the resource was
+	// created from the AWS IAM Roles Anywhere Integration.
+	OriginIntegrationAWSRolesAnywhere = common.OriginIntegrationAWSRolesAnywhere
```

Comment:
is the constant needed in both packages for some reason? oh i guess we have a bunch of constants like this 🤔 seems weird but feel free to ignore this comment

Reply:
I added it for the sake of consistency.
I think we could just have them set in the `api/types/common` package, but removing the existing ones is tricky because it breaks backwards compat.